abstractions/php/src/Serialization/ParseNode.php

```diff
@@ -5,6 +5,8 @@
 
 use DateTimeInterface;
 use Closure;
+use Microsoft\Graph\Core\Models\Date;
```

Comment:
those models should be under kiota abstractions and not graph core

Reply:
Fixed that in the latest commit